### PR TITLE
add the calssification catagory of test case (farEdge Telco NonTelco)

### PR DESCRIPTION
**Pull request by aabughosh**

### CODE REVIEW

**Review by shimritproj — APPROVED** [review]

File: pkg/claim/catalog.go [hunk 1]
```diff
@@ -16,6 +16,21 @@ import (
...
 )
 
+/*
+{
+	"ForTelco": "Mandatory",
+	"FarEdge" : "Optional",
+	"ForNonTelco": "Optional",
+	"ForVZ": "Mandatory"
+   }*/
+// struct for calssification
```
Comment (sebrandon1) [inline pkg/claim/catalog.go:26]:
```suggestion
// struct for classification
```

File: pkg/claim/catalog.go [hunk 2]
```diff
@@ -16,6 +16,21 @@ import (
...
+type CatagoryClassification struct {
```
Comment (sebrandon1) [inline pkg/claim/catalog.go:27]:
```suggestion
type CategoryClassification struct {
```

File: pkg/claim/catalog.go [hunk 3]
```diff
@@ -41,13 +56,15 @@ type TestCaseDescription struct {
 
 	// Whether or not automated tests exist for the test case. Not to be rendered.
 	Qe bool `json:"qe" yaml:"qe"`
+	// calassification for each test case
+	Catagoryclassification CatagoryClassification `json:"Catagoryclassification" yaml:"Catagoryclassification"`
```
Comment (sebrandon1) [inline pkg/claim/catalog.go:60]:
```suggestion
	// classification for each test case
	Categoryclassification CategoryClassification `json:"Categoryclassification" yaml:"Categoryclassification"`
```

**Review by sebrandon1 — CHANGES_REQUESTED** [review]
Typos!

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**